src/test/scala/org/hammerlab/guacamole/commands/jointcaller/SomaticJointCallerSuite.scala

```diff
@@ -75,13 +73,16 @@ class SomaticJointCallerSuite extends GuacFunSuite {
       .toMap
 
     calls(("chr1", 179895860)).singleAlleleEvidences.length should equal(1)
-    calls(("chr1", 179895860)).bestAllele.isSomaticCall should be(false)
-    calls(("chr1", 179895860)).bestAllele.isGermlineCall should be(true)
-    calls(("chr1", 179895860)).bestAllele.allele.ref should equal("T")
-    calls(("chr1", 179895860)).bestAllele.allele.alt should equal("C")
-    calls(("chr1", 179895860)).bestAllele.allEvidences.head.annotations.get.strandBias.phredValue should equal(0)
-    calls(("chr1", 179895860)).bestAllele.allEvidences.head.annotations.get.annotationsFailingFilters should equal(Seq.empty)
-    calls(("chr1", 179895860)).bestAllele.annotations.get.annotationsFailingFilters should equal(Seq.empty)
+
+    val bestAllele = calls(("chr1", 179895860)).bestAllele()
+
```

Comment:
hm, I disagree; there is extensive parallel structure in the block below that this line doesn't match, logically or visually :-\